src/main/java/com/linkedin/kmf/services/ClusterTopicManipulationService.java

```diff
@@ -45,22 +53,33 @@
   private final ScheduledExecutorService _executor;
   private final AdminClient _adminClient;
   private boolean _isOngoingTopicCreationDone;
+  private boolean _isOngoingTopicDeletionDone;
   private final AtomicBoolean _running;
   private String _currentlyOngoingTopic;
   int _expectedPartitionsCount;
   // TODO -- ClusterTopicManipulationMetrics implementation in progress!
-  private ClusterTopicManipulationMetrics _clusterTopicManipulationMetrics;
+  private final ClusterTopicManipulationMetrics _clusterTopicManipulationSensors;
```

Comment:
Class type is Metrics but variable is Sensors, kind of confusing here

Reply:
renamed